opendoors/open-doors-tables-working.sql

```diff
@@ -114,6 +114,7 @@ CREATE TABLE IF NOT EXISTS `stories`
     `do_not_import` tinyint(1)   NOT NULL DEFAULT '0',
     `ao3_url`       varchar(255)          DEFAULT NULL,
     `import_notes`  varchar(1024)         DEFAULT '',
+    `rating` 		tinyint(4)	 NOT NULL DEFAULT 0,
```

Comment:
I don't think we actually store anything in this field in the eFiction code (or we'd have noticed it was missing sooner!), so this might as well start off life as it will eventually end it, as a varchar big enough to hold `Teen And Up Audiences` (ie [the Archive's 5 official ratings](https://archiveofourown.org/help/symbols-key.html))

Reply:
Yeah, I see how I didn't think this through all the way. Should there be a default rating?

Reply:
I think it's fine to leave it empty as having a NULL in the field will make it clearer if there's a processing error further down the process when we want to put the original ratings in.